src/services/ui/src/components/RHF/SlotField.tsx

```diff
@@ -113,6 +119,18 @@ export const SlotField = ({
         </Select>
       );
     }
+    case "Multiselect": {
+      const options = props?.options as MultiselectOption[];
+      const value = field.value as string[];
+
+      return (
+        <Multiselect
```

Comment:
Are we good to not forward any props here besides options?

Reply:
Excellent catch! You're right, we may need to forward props like `className` to the Multiselect. I've added it.